docs/operations/images.md

```diff
@@ -87,8 +87,18 @@ aws ec2 describe-images --region us-east-1 --output table \
 
 ### CentOS 8
 
-The CentOS Project doesn't provide any official images in AWS at the moment.
-Please [report](https://github.com/kubernetes/kops/issues/new/choose) any changes.
+CentOS 8 is based on Kernel version **4.18** which fixes some of the bugs present in RHEL/CentOS 7 and effects are less visible.
+
+One notable change is the addition of `iptables` NFT, which is the only iptables backend available. This may not be supported by some CNI plugins and should be used with care.
+
+Available images can be listed using:
+
+```bash
+aws ec2 describe-images --region us-east-1 --output table \
+  --owners 679593333241 \
```

Comment:
So this is the account ID that is kops' `WellKnownAccountCentOS` but the account ID in the CentOS wiki is different: https://wiki.centos.org/Cloud/AWS It would be nice to figure out why they're different and if we should switch to the one mentioned upstream.

Reply:
This is the AWS Marketplace account. Changed it a bit and now references the `AWS Provided Marketplace Images with Updates`:
https://wiki.centos.org/Cloud/AWS#AWS_Provided_Marketplace_Images_with_Updates

We could move to their new owner ID and images, but don't know if there's a lot of value and we would have to change the WellKnownAccountCentOS also.

Reply:
Thought about it a  bit more and just changed it to the new CentOS repo. Easier to understand for anyone and probably will get the latest updates.